server/database/pool.js

```diff
@@ -0,0 +1,19 @@
+const { Pool } = require('pg');
+require('dotenv').config()
+let databaseName;
+
+if (process.env.NODE_ENV == 'test') {
+  databaseName = process.env.TEST_DATABASE_NAME;
```

Comment:
So ideally I like to have the same environment variable names in each in environment.

So the only decision point is: which environment are we in, then load the relevant set of environment variables (I know this goes agains the dotenv documentation but I think they're not being realistic).

Reply:
Thanks Matt. I was experiencing issues loading variables from different env files when the environment change. However, I'll look into this further, to implement what you've suggested.